db.go

```diff
@@ -13,7 +13,7 @@ type Migratable interface {
 type Postgres struct{}
 
 func (p Postgres) SelectMigrationTableSql() string {
-	return "SELECT tablename FROM pg_catalog.pg_tables WHERE tablename = $1"
+	return "SELECT table_name FROM information_schema.tables WHERE table_name= $1 AND table_catalog = $2"
```

Comment:
This confuses me because I thought the intent of the pull request is to have variable schemas (http://www.postgresql.org/docs/8.2/static/ddl-schemas.html) under a constant database.